### DISCUSSION TITLE
How to set Default Action

### DISCUSSION THREAD

**Discussion by lordofscripts** [dunst-project/dunst#1360] [Q&A] [answered]
I am experimenting with programmatic notifications with GO, not with `dunstify`.

I set up my test notifications with two actions: Cancel & Open. My program is capable of receiving the **Action** request from the server. When the notifications appear on the screen, I _see_ they have the `(A)` indicators. Therefore the `Dunst` server has properly recognized them in my notifications.

My question is, **How do I specify which of the two actions is** the _default action_?

I have read a lot of documentation, browsed code (shouldn't be a need for that as user) and I don't see where it indicates how to to that.

**Comment by bynect** [comment, marked as answer]
The action name should be 'default'. Note that the action name and label can be different. For example using dunstify

dunstify --action="default,Reply" --action="forwardAction,Forward" "Message Received"  'Hello'

**Comment by lordofscripts** [comment]
Great explanation, thanks!
